client/blocks/nps-survey/index.jsx

```diff
@@ -49,49 +51,57 @@ class NpsSurvey extends Component {
 	}
 
 	render() {
+		const { translate } = this.props;
+
 		const className = classNames( 'nps-survey', {
 			'is-recommendation-selected': Number.isInteger( this.state.score ),
 			'is-submitted': this.props.hasAnswered,
 		} );
 
 		return (
-			<div className={ className }>
+			<Card className={ className }>
 				<div className="nps-survey__question-screen">
-					<div>How likely is it that you would recommend WordPress.com to your friends, family, or colleagues?</div>
-					<div>
+					<div className="nps-survey__question">
+						{ translate( 'How likely are you to recommend WordPress.com to your friends, family, or colleagues?' ) }
+					</div>
+					<div className="nps-survey__recommendation-select-wrapper">
 						<RecommendationSelect
 							value={ this.state.score }
 							disabled={ this.props.hasAnswered }
 							onChange={ this.handleRecommendationSelectChange }
 						/>
 					</div>
-					<div>
+					<div className="nps-survey__buttons">
 						<Button primary
 							className="nps-survey__finish-button"
 							disabled={ this.props.hasAnswered }
 							onClick={ this.handleFinishClick }
 						>
-							Finish
+							{ translate( 'Finish', { context: 'verb' } ) }
```

Comment:
👋 
You can use `Finish` with no context. It's mostly not ambiguous, and we've already got that translated.
(The a8ci18n bot comment here is somewhat broken, but you can see the proper suggestion here: https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/nps-survey-styles)